src/pages/Task.tsx

```diff
@@ -98,6 +98,7 @@ const TaskCore: React.FC = () => {
 
   useNetworkStatus(startPolling, stopPolling);
   const task = get(data, "task");
+  const annotation = get(task, "annotation");
   const canAbort = get(task, "canAbort");
```

Comment:
instead of using `get()` can you  use `task?.field` like on line 103 That way we can maintain some type safety. Alternatively you can try to destructure all the fields from the task object.